src/components/detail/DetailTitle.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import './DetailTitle.scss';
+
+const DetailTitle = ({
+  tags,
+  korName,
+  engName,
+  description,
+  colors,
+  volume,
+  price,
+}) => {
+  //setImgSrc([images[0]]);
+  const [dropDown, setDropDown] = useState(false);
+  const [colorIdx, setColorIdx] = useState(0);
+  const handleDropDown = e => {
+    dropDown ? setDropDown(false) : setDropDown(true);
+  };
```

Comment:
event 객체는 인자로 받지 않아도 되겠네요! 그리고 다음처럼 수정할 수 있을 것 같습니다.
```suggestion
  const handleDropDown = () => {
    setDropDown(!dropDown);
  };
```

Reply:
필요없는 이벤트 객체는 지우고 다음처럼 수정할수 있는 것들은 수정했습니다.